### DISCUSSION TITLE
Multilingual home pages display wrong content

### DISCUSSION THREAD

**Discussion by grandixximo** [requarks/wiki#3696] [Error / Bug Report]
**Describe the bug**
I enabled Multilingual Namespacing for 2 languages en and zh 
I created my english home in /en/home as a markdown page type and my chinese home in /zh/home also as a markdown page type
When i switch between /en/home and /zh/home the chinese home page displays the content of the english home page, when editing the /zh/home the edit content is displayed in Chinese as i wrote it, upon saving i can see the /zh/home displaying chinese contect, but after switching locale the chinese home displays content in english again

**Expected behavior**
Switching locale displayd the correct language content

**Host Info (please complete the following information):**
 - OS: Ubuntu 18.04
 - Wiki.js version: 2.0.1
 - Database engine: [PostgreSQL 10.10]

**Comment by grandixximo** [comment]
This is still the behavior with the last update, why are other languages homes not reachable?

**Comment by grandixximo** [comment]
could an option be made to visualize different locales home pages?

**Comment by ba-moun** [comment]
Maybe the same problem: when I choose a language other than english (i.e. french) and I click on a page in the main menu (i.e. fr/folder/page), I'm redirected to the english root (i.e. en/folder/page).

**Reply by krolsekat** [reply]
I'm having this same issue with the home button.
We're using AAD login combined with local groups in Wiki.js
Users are assigned (mostly) to one group and all groups have a language-specific redirect on login.
When signing in for the first time we will be redirected to **/en/home** which is a restricted page for some users. When clicking on **Login as** and signing in with the same credentials the redirection is correct.
This also happens after signing out and signing back in again.

**Comment by ba-moun** [comment]
The home button in the page path (upon the page content) redirect to the principal language home page whatever the language choosed.
@NGPixel, do you confirm that this message and the previous are in the same ticket?

**Comment by grandixximo** [comment]
Any fix coming for this?
